imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -146,7 +146,8 @@ class CaseExplorer extends Component {
     const { statusFilterValues, roleFilterValues, sortBy, open } = this.state
     if (isLoading) return <Preloader />
     const caseGrouping = this.makeCaseGrouping(caseList, statusFilterValues, roleFilterValues, sortBy, allNotifications, unreadNotifications)
-    const cases = caseGrouping.sort(sorters[sortBy])
+    const defaultList = caseGrouping.sort(sorters[SORT_BY.DATE_ASCENDING])
```

Comment:
@sophieyang0813 are you sure you need to sort Ascending if you need to sort from newest to oldest? 🤔